docs/stories/WebTools.md

```diff
@@ -0,0 +1,53 @@
+# Web Tools
+
+## Configure web application directory in WTP configurations for web application projects
+
+### Motivation
+The `ide-integration.md` document in Gradle core specifies a story ["Expose web application directory"](...). The goal is to define an initial end-to-end framework for Buidship/WTP integration, and to
+communicate the location of the web application directory to WTP.
+
+### Estimate
+
+- ? day(s)
+
+### Implementation
+- Create branch in buildship repository `wtp-integration`.
+- Define `org.eclipse.buildship.wtp` plugin in `wtp-integration`.
+- Define `org.eclipse.buildship.wtp.test` plugin in `wtp-integration`.
+- Wait for corresponding story in TAPI to be completed.
+- Upgrade Tooling Commons (if necessary).
+- Define configuration class in `org.eclipse.buildship.wtp`, either configurator class, or a similar class to `DefaultWorkspaceGradleOperations`.
+    - Retrieve Web application model from TAPI
+    - If a project is a web application project (model will be `null` if project doesn't apply `war` plugin)
+        - Add Java and Dynamic Web Facets to project, which will generate component file
+        - Configure web app dir location in component file
+    - Otherwise, do nothing
+- Invoke WTP configuration from Buildship core
+
+### Test cases
+- Configuration only invoked on web application projects
+- Java facet is added to project
+- Dynamic Web Facet is added to project
+- If Java Facet already on project but has incorrect version, version is updated to correct version based on source Java version
+- If Dynamic Web Facet already on project but has incorrect version, version is updated to correct version
```

Comment:
> It's implemented in the https://github.com/Ian-Stewart-Binks/buildship-wtp/blob/master/org.eclipse.buildship.wtp.core/src/main/java/org/eclipse/buildship/wtp/core/configurator/WebApplicationConfigurator.java#L188

Yes, just trying to get what was actually going on rather than the implementation. That bit was more copy + paste from m2e.wtp when I asked the question last time, and I want to make sure I know what I'm doing for this rather than how to do it.